docs/synapse/userguides/storm_ref_cmd.rstorm

```diff
@@ -1042,23 +1267,61 @@ The ``model.edge.get`` command allows you to retrieve all of the keys that have
 model.edge.del
 ++++++++++++++
 
-The ``model.edge.del`` command allows you to delete the key from a light edge (such as a ``doc`` property to specify a definition for the light edge). Deleting a key from a specific light edge does not delete the key from Synapse (e.g., the property can be re-added to the light edge or to other light edges).
+The ``model.edge.del`` command allows you to delete the key from a light edge (such as a ``doc`` property
+to specify a definition for the light edge). Deleting a key from a specific light edge does not delete
+the key from Synapse (e.g., the property can be re-added to the light edge or to other light edges).
 
 **Syntax:**
 
-
 .. storm-cli:: model.edge.del --help
 
 
+.. _storm-movenodes:
+
+movenodes
+---------
+
+The ``movenodes`` command allows you to move nodes between layers (:ref:`gloss-layer`) in a Cortex.
+
+The command will move the specified storage nodes (:ref:`gloss-node-storage`) to the target layer. If
+a node is the "left hand" node (``n1``) of two nodes joined by a light edge (``n1 -(*)> n2``), then
+the edge is also moved.
+
+Nodes are fully removed from the source layer(s) and added to (or merged with existing nodes in)
+the target layer. If intervening layers exist between the source and target, the nodes are moved
```

Comment:
For this command, the source layer(s) and destination layer specified are the only layers involved in the operation. By default the source layers are "everything except the top layer in the view", but if there were multiple layers underneath and you only specified the bottom one, anything in those middle layers would be ignored.

Reply:
Revised wording.